templates/configmap.yaml

```diff
@@ -92,36 +92,14 @@ data:
     tier = airflow
     component = worker
     release = {{ .Release.Name }}
-    platform = {{ .Values.platform.release }}
-    workspace = {{ .Values.platform.workspace }}
-{{- end }}
-
-    [astronomer]
-{{- if .Values.webserver.jwtSigningCertificateSecretName }}
-    jwt_signing_cert = {{ template "airflow_webserver_jwt_cert_path" . }}
-    jwt_audience = {{ printf "%s/%s" ( include "deployments_subdomain" . ) .Release.Name }}
+    {{- range $key, $value := .Values.labels }}
+    {{ printf "%s = %s" $key $value }}
+    {{- end }}
 {{- end }}
 
+{{- if .Values.webserver.webserverConfig }}
   webserver_config.py: |
-    import os
-    from airflow import configuration as conf
-    from flask_appbuilder.security.manager import AUTH_REMOTE_USER
-    basedir = os.path.abspath(os.path.dirname(__file__))
-
-    # The SQLAlchemy connection string.
-    SQLALCHEMY_DATABASE_URI = conf.get('core', 'SQL_ALCHEMY_CONN')
-
-    # Flask-WTF flag for CSRF
-    CSRF_ENABLED = True
-
-    # ----------------------------------------------------
-    # AUTHENTICATION CONFIG
-    # ----------------------------------------------------
-{{- if .Values.webserver.jwtSigningCertificateSecretName }}
-    AUTH_TYPE = AUTH_REMOTE_USER
-
-    from astronomer.flask_appbuilder.security import AirflowAstroSecurityManager
-    SECURITY_MANAGER_CLASS = AirflowAstroSecurityManager
+    {{- .Values.webserver.webserverConfig | nindent 4 }}
 {{- end }}
 
 {{- if .Values.scheduler.airflowLocalSettings }}
```

Comment:
We may want to think about removing our `pod_mutation_hook` settings, we can pass `airflow_local_settings` from Astro chart probably ?

Reply:
https://github.com/astronomer/airflow-chart/pull/66/files#diff-1d2f4c201ce0c92586e4e3c5f9a6e03eR109-R130

Reply:
I'm fine with removing it, maybe leave something in there commented out as an example? I do feel like this specific feature is being used in the wild a lot, so it might be nice to have a little example in there.

Reply:
Yeah I agree., something like:

```yaml
# 1) A "policy" function that has the ability to mutate task attributes based on other task or DAG attributes.
# 2) "pod_mutation_hook". This could be used, for instance, to add sidecar or init containers
#     to every worker pod or just add labels to all pods launched by KubernetesExecutor or KubernetesPodOperator.

#     def pod_mutation_hook(pod: Pod):
#         extra_labels = {
#             "kubernetes-executor": "False",
#             "kubernetes-pod-operator": "False"
#         }
#         if 'airflow-worker' in pod.labels.keys() or \
#                 conf.get('core', 'EXECUTOR') == "KubernetesExecutor":
#             extra_labels["kubernetes-executor"] = "True"
#         else:
#             extra_labels["kubernetes-pod-operator"] = "True"
#         pod.labels.update(extra_labels)
```

Reply:
What do you think about the above command